app/main.py

```diff
@@ -1,2 +1,75 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    key_hash: int
+
+
 class Dictionary:
     pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` is calculated as `2 / 3`, which results in a float value (approximately 0.6667). If the intention was to use a fraction, consider using `Fraction(2, 3)` from the `fractions` module to maintain precision.